FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Statistics.java

```diff
@@ -13,21 +13,22 @@ public enum Statistics {
     TOTAL,
     PERCENTILE,
     CUMULATIVE,
+    NDAYCUMULATIVE,
     MAXAVERAGEDURATION,
     MAXACCUMDURATION,
     PLOTTINGPOSITION,
     COMPUTABLE,
     UNDEFINED;
 
-    public static String pack(EnumSet<Statistics> set){ //returns a list of the statistics in a string format
-        String ret = "";
+    public static String pack(Set<Statistics> set){ //returns a list of the statistics in a string format
```

Comment:
Is this method needed? It does not appear to be called.

Reply:
I think this is in case someone wanted to make an API call to see the list of available statistics